src/hermes/settings.py

```diff
@@ -0,0 +1,121 @@
+from pydantic import BaseModel
+import toml
+import pathlib
+from typing import Any, Dict, Tuple, Type, ClassVar
+
+from pydantic.fields import FieldInfo
+
+
+from pydantic_settings import (
+    BaseSettings,
+    PydanticBaseSettingsSource,
+    SettingsConfigDict,
+)
+
+
+class HarvestCff(BaseModel):
+    enable_validation: bool
+
+
+class HarvestSettings(BaseModel):
+    sources: list[str]
+    cff: HarvestCff = HarvestCff
+
+
+class DepositTargetSettings(BaseModel):
+    site_url: str
+    communities: list[str] = None
+    access_right: str
+
+
+class DepositSettings(BaseModel):
+    target: str
+    invenio: DepositTargetSettings = DepositTargetSettings
+    invenio_rdm: DepositTargetSettings = DepositTargetSettings
+
+
+class HermesSettings(BaseSettings):
+
+    model_config = SettingsConfigDict(env_file_encoding='utf-8', extra='allow')
+
+    harvest: HarvestSettings = HarvestSettings
+    deposit: DepositSettings = DepositSettings
+    config_path: ClassVar[pathlib.Path] = "hermes.toml" #TODO : Set config Path in cli
+
+    def __init__(self, conf_path):
+        super().__init__()
+        self.__class__.config_path = conf_path
```

Comment:
Forgot in my review: Why is this set on class-level? Are you sure this is what want? (Well, I can imagine why this might be a good idea, but I cannot read it in any documentation yet... ;) )